### DISCUSSION TITLE
posts/2020/12/eleventy-hugo-comparing-contrasting/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#40] [Announcements]
# posts/2020/12/eleventy-hugo-comparing-contrasting/

Trying to choose between the two best static site generators? See if these points address your personal hot buttons.

https://www.brycewray.com/posts/2020/12/eleventy-hugo-comparing-contrasting/

**Comment by holliebutler** [comment]
This was incredibly helpful to me, thank you!

**Reply by brycewray** [reply]
@holliebutler You’re very welcome! You might also want to check my more recent post on this subject:

https://www.brycewray.com/posts/2023/02/which-ssgs-recommend/
